utils/checks.stanza

```diff
@@ -64,8 +67,8 @@ public pcb-check within? (t:Toleranced, value:Toleranced) :
     subcheck-description = "Check that a value is within a given range"
     pass-message =         "Value %_ to %_ is within the range of %_ to %_" % [min-value(t) max-value(t) min-value(value) max-value(value)]
     fail-message =         "Value %_ to %_ is not within the range of %_ to %_" % [min-value(t) max-value(t) min-value(value) max-value(value)]
-    locators =             [t value]
     )
+    ; locators =             [t value]
```

Comment:
Locators aren't yet supported in the new macro, so I have commented them out in the meantime!

Reply:
Gotcha